perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -131,7 +190,11 @@ def _GetOptionDecoderConstructions(cls):
     """
     result = super(CustomMachineTypeSpec, cls)._GetOptionDecoderConstructions()
     result.update({'cpus': (option_decoders.IntDecoder, {'min': 1}),
-                   'memory': (MemoryDecoder, {})})
+                   'memory': (MemoryDecoder, {}),
+                   'gpus': (GceGpuDecoder, {
```

Comment:
Is it a requirement that gpus must only be attached to custom machine types or can they be attached to standard/highcpu/highmem types?